CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ changes.
 
 - HeadLogic Outcome is now being trace on every protocol step transition.
 
+- Check commits are reimbursed in nu head.
```

Comment:
Let's call it `head validator`?

This is a `**BREAKING**` change as all other script changes.